### DISCUSSION TITLE
Would your country benefit from having a self service quest for fuel stations?

### DISCUSSION THREAD

**Discussion by naposm** [streetcomplete/StreetComplete#3128] [Q&A]
Hi! I need your help! I have one meta-quest for you:

If you are from a country different from:
- Italy
- Germany
- Australia

please let me know if you think this quest could be useful for you, or it would be useless and spammy since the answer would always be the same.

We're trying to understand which countries would benefit from having a full service/self service tag on their fuel stations to implement this PR: #2822 .

Thank you very much for your attention! 🤗

**Comment by mnalis** [comment]
In my limited experience (I'm not a car driver) I would say in Croatia you practically always fill you own tanks (for regular petrol/diesel at least, don't know about LPG etc), so it would be [self_service=only](https://wiki.openstreetmap.org/wiki/Key:self_service), I guess? Or `self_service=yes`? 

However I've done http://overpass-turbo.eu/s/1a5y and it does show few marked as `self_service=no` in Croatia, so they might exist. (**or** they might be incorrectly mapped)

Maybe because after filling the tank yourself you practically always go to pay at the counter. So it's [self_checkout=no](https://wiki.openstreetmap.org/wiki/Key:self_checkout), but is it still `self_service` or does it make not being so? So confusing.

I also do not know what `full_service=*` even means, it's [not documented](https://wiki.openstreetmap.org/wiki/Key:full_service) and [amenity=fuel service section](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dfuel#Service) is also confusing?

What about if there are (in summer only usually) students (I guess not officially employed, but allowed by owners, that try to make a few bucks) that offer to wash your windshield/car? That sometimes happens. Is it then `self_service=no`? or `full_service=yes`? Or both? Something in between?

Could we at least have a clean definition on the [wiki](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dfuel#Service) what is actually being mapped, and what the tags mean exactly, **_before_** we start mapping them? Discussion in https://github.com/streetcomplete/StreetComplete/issues/1827 did not seem conclusive to me?

**Comment by peternewman** [comment]
The vast majority of UK petrol stations are self-service.

I've also driven through a lot of mainland Europe over the years (France, Spain, Belgium, Luxembourg, Croatia, Germany, Austria, Slovenia, a little bit of Italy) and I don't particularly remember much other than self-service in those places. That may just be my memory failing, but I'd imagine it would stand out as quite a novelty given it's so unheard of in the UK.

**Comment by matkoniecz** [comment]
In Poland nearly all stations are self service by default, many have signposted option to call for assistance (either implicitly or explicitly for disabled people).

In Poland this quest would not make sense.

I had minimal experience with Slovenia and Croatia (hitchhiking) and based on tiny sample - self service appeared to be normal.

In USA it depends on state - with some outlawing self service and in some it being normal.
